cmd/cmd_utils.go

```
@@ -543,3 +546,122 @@ func CheckForAtmosUpdateAndPrintMessage(atmosConfig schema.AtmosConfiguration) {
+			})
+			cmd.SetUsageTemplate(template)
+			cc.Init(&cc.Config{
+				RootCmd:  cmd,
+				Headings: cc.HiCyan + cc.Bold + cc.Underline,
+				Commands: cc.HiGreen + cc.Bold,
+				Example:  cc.Italic,
+				ExecName: cc.Bold,
+				Flags:    cc.Bold,
+			})
+		}
+		cmd.Help()
```

golangci-lint (1.62.2)

Error return value of `cmd.Help` is not checked

(errcheck)